app/src/main/res/values/strings.xml

```diff
@@ -381,5 +382,16 @@
     description="The close button text of the dialog shown when a user's pre-release app has expired.">
     Close app
   </string>
+  <string
+    name="ratio_content_description_separator"
+    description="The separator that would be used to build content description for ratio interaction">
```

Comment:
I suggest providing an example here for additional context, too. Also, suggest specifying that this is the text that will be read out for screenreaders for submitted ratio answers. Content description is a develop term.

Reply:
done